addons/mail/static/src/utils/common/dates.js

```diff
@@ -4,6 +4,9 @@ const { DateTime } = luxon;
  * @param {luxon.DateTime} datetime
  */
 export function computeDelay(datetime) {
+    if (!datetime) {
+        return 0;
+    }
```

Comment:
I don't like this change, the method shouldn't be called if there's no datetime.
Worse case it should return undefined rather than 0.

Reply:
Probably related to slot issue we fixed in recent PR?